hacker_Rank/May11_Answer.py

```diff
@@ -0,0 +1,21 @@
+from algorithm import Algo
+
+obj = Algo()
+
+
+def solutuion(a):
+    max_value = obj.max_value(a)
+    series = set(item for item in range(1, max_value + 1))
+    a = set(a)
+    difference = series.difference(a)
```

Comment:
``` a = set(a) ``` <-- is this required?, did you try without using?
``` series.difference(a) ``` will not work ?

Reply:
okay i will check